### PR TITLE
Getting started

### LINKED ISSUE #5: Add one-line getting started command to build+run tests

### PR DESCRIPTION
**Pull request by pickfire**
Fix #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,23 @@ that includes a link to your comment.
...
 
+Getting Started
+---------------
+
+Cargo and rustc are a prerequisite, to install check https://rustup.rs/.
+
+To build and run tests.
+
+```fish
+> cargo test
```
Comment (scooter-dangle) [inline README.md]: With the `>`, it's not actually valid fish syntax. (Could also leave the `>` but remove the `fish` code-block annotation.

```suggestion
cargo test
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,23 @@ that includes a link to your comment.
...
+```
+
+To run the command. (Not working as of now)
+
+```fish
+> cargo run
```
Comment (scooter-dangle) [inline README.md]: Same as previous comment, `> ` would make this invalid fish syntax
```suggestion
cargo run
```

File: README.md (hunk 1 above)
Comment (pickfire) [reply README.md]: `>` is supposed to be the fish prompt, should we remove that?

Comment (scooter-dangle) [reply README.md]: Yeap (if you're describing valid fish syntax).

Comment (pickfire) [reply README.md]: So I assume this is resolved?

**Comment by pickfire** [comment]
I believe the mention can be another commit but I feel lazy adding that.

File: README.md (hunk 1 above)
Comment (scooter-dangle) [reply README.md]: Looks like the `>` is still there? Will go ahead and remove them + merge.

Comment (pickfire) [reply README.md]: Ah, I did not know you want me to remove them. Sorry about that.

**Pull request merged**